src/features/party/PartiesProvider.tsx

```diff
@@ -51,6 +51,25 @@ export function useCurrentPartyQueryDef(enabled: boolean) {
   };
 }
 
+export function useCurrentPartyRolesQueryDef(enabled: boolean) {
+  const { fetchRoles } = useAppQueries();
+  return {
+    queryKey: ['fetchCurrentPartyRoles', enabled],
+    queryFn: fetchRoles,
+    enabled,
+  };
+}
```

Comment:
Kan vi bruke `queryOptions`  fra tanstack her, så får den riktig type? Og blir da litt som queryFactories som jeg snakket om i tanstack-presentasjonen 😄 
```suggestion
  return queryOptions({
    queryKey: ['fetchCurrentPartyRoles', enabled],
    queryFn: fetchRoles,
    enabled,
  });
}
```